spacy/tokens/doc.pyx

```diff
@@ -746,6 +747,10 @@ cdef class Doc:
         # Return the merged Python object
         return self[start]
 
+    def print_tree(self, light=False, flat=False):
+        """Returns the parse trees in the JSON (Dict) format."""
+        return parse_tree(self, light=light, flat=flat)
```

Comment:
@honnibal Is this close to what u had in mind? Added a method for `Doc` to `print_tree`. In essence this line should pass a `doc` object to the `parse_tree` method of the printers module (see the slightly different usage there).

However on running the test I got the error below, not sure what's the issue.
```shell
EN = <spacy.en.English object at 0x10ff6ab38>

    def test_parse_tree(EN):
        text = 'I like New York in Autumn.'
        EN = English(parser=False)
        doc = EN(text, tag=True)
        doc.from_array([HEAD], numpy.asarray([[1, 0, 1, -2, -3, -1, -5]], dtype='int32').T)
        # full method parse_tree(text) is a trivial composition
>       trees = doc.print_tree()

spacy/tests/tokens/test_tokens_api.py:217: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
spacy/tokens/doc.pyx:655: in spacy.tokens.doc.Doc.print_tree (spacy/tokens/doc.cpp:13233)
    return parse_tree(self, light=light, flat=flat)
spacy/tokens/printers.py:54: in parse_tree
    return [POS_tree(sent.root, light=light, flat=flat) for sent in doc_clone.sents]
spacy/tokens/printers.py:54: in <listcomp>
    return [POS_tree(sent.root, light=light, flat=flat) for sent in doc_clone.sents]
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   "to install the data" % self.vocab.lang)
E   AttributeError: 'NoneType' object has no attribute 'lang'
```